rdt-open-reader/src/main/java/io/ona/rdt/activity/PatientRegisterActivity.java

```diff
@@ -217,14 +230,7 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
             }
         });
 
-        String latestSyncTimestamp = allSharedPreferences.getPreference(Constants.Preference.CTS_LATEST_SYNC_TIMESTAMP);
-        if (StringUtils.isNotBlank(latestSyncTimestamp)) {
-            Date lastSyncDate = new Date(Long.parseLong(latestSyncTimestamp));
-            String lblSync = getString(R.string.lbl_latest_sync);
-            TextView tvLatestSyncDate = findViewById(R.id.tv_latest_sync_date);
-            tvLatestSyncDate.setText(String.format(lblSync, new SimpleDateFormat(LATEST_SYNC_DATE_FORMAT, Locale.getDefault()).format(lastSyncDate)));
-            tvLatestSyncDate.setVisibility(View.VISIBLE);
-        }
+        updateSyncDate(allSharedPreferences);
```

Comment:
Do we still need this here?

Reply:
I think it's needed because whenever the app will be launched then we will need to show the last sync time

Reply:
The first time on install, we don't need it because there's no initial sync yet. Subsequently, won't the previous value be persisted? Have you verified it doesn't work?

Reply:
yeah, I think we don't need to call this method when the activity will launch. Now it's removed